include/psa/crypto_sizes.h

```diff
@@ -1086,4 +1086,33 @@
 #define PSA_CIPHER_FINISH_OUTPUT_MAX_SIZE           \
     (PSA_BLOCK_CIPHER_BLOCK_MAX_SIZE)
 
+/** A sufficient output buffer size for psa_pake_output().
+ *
+ * If the size of the ciphertext buffer is at least this large, it is
+ * guaranteed that psa_pake_output() will not fail due to an
+ * insufficient ciphertext buffer size. The actual size of the output might be
+ * smaller in any given call.
+ *
+ * See also #PSA_PAKE_OUTPUT_MAX_SIZE
+ *
+ * \param alg           A PAKE algorithm (PSA_ALG_XXX value such that
+ *                      #PSA_ALG_IS_PAKE(\p alg) is true).
+ * \param cipher_suite  A cipher suite that is compatible with algorithm \p alg.
+ * \param output        An output type used with algorithm \p alg.
```

Comment:
`output` is usually an output buffer. What's an “output type”? A `psa_pake_data_t`?